config/routes.rb

```diff
@@ -0,0 +1,19 @@
+Rails.application.routes.draw do
+  get 'users/index'
+  get 'users/create'
+  get 'users/new'
+  get 'users/show'
+  get 'users/destroy'
```

Comment:
I don't think you need all these routes for `users` - only `index` and `show` are needed for this project.